native-embedding/src/test/c/evaljs.c

```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+#include "nativejavavm.c"
```

Comment:
Please don't `#include` C files. This is a recipe for disaster, especially with global variables defined in the C file. This might work for these tiny toy examples, but in a real project, you're going to get duplicate symbols and linker errors like this. People are going to copy from this example, so we should do it in a way that works in the real world.

Write a proper header file for it with function prototypes and `extern` definitions for the globals, include the header file, compile the C file separately and link it.